### DISCUSSION TITLE
Performance of javascript BirdNET model?

### DISCUSSION THREAD

**Discussion by tphakala** [Mattk70/Chirpity-Electron#152] [General]
How is performance of javascript BirdNET model, does it perform well enough to use on edge devices like modern smartphones etc?

**Comment by Mattk70** [comment]
Hey, I guess that depends on the use case. I know @kahst and his team built a proof of concept web app using the JS port of BirdNET. It worked, although for that app, they had a live spectrogram and I suspect the competing demands on the Gpu (prediction and canvas rendering) made the spec display jamky.  I didn’t look at it from my phone.
